### DISCUSSION TITLE
Metadata

### DISCUSSION THREAD

**Discussion by quebecman** [RandomEngy/VidCoder#842] [General]
Hello David,

  Hmmmm.. I have probably missed this option that was added in the PICKER, but can I know what it does? 

As my understanding, this is suppose to pass the information somewhere in Mediainfo or something like it.

Thanx.
Regards.

**Comment by RandomEngy** [comment]
It passes through information like release date, director, actors, etc. That metadata would probably show up in Mediainfo.
